index.html

```diff
@@ -8,6 +8,54 @@
   <link rel="stylesheet" href="styles.css">
```

Comment:
### I didn't have access to the README file from this branch file changes so I'll just leave some comments here:

- [ ] Since the live demo is not a mandatory option at least for this Milston please consider removing it, you will have the chance to make it back in coming lessons 😄🚀
- [ ] **[OPTIONAL]** the FAQ section is optional if you don't have anything to provide please consider removing it instead, but providing some information is recommended👍